### ISSUE TITLE
Win 7/8 auto-gpu detection

### ISSUE THREAD

**Issue by MaynardMiner** [MaynardMiner/SWARM#123]
There is an issue regarding auto-gpu detection in win 7/8.

I am working on solution.

**Comment by petrovsky34** [comment]
I will try to reinstall the drivers to do

**Comment by petrovsky34** [comment]
![Снимок4444444](https://user-images.githubusercontent.com/39003568/57144626-1da9cf00-6dca-11e9-8407-3017c070b0c8.PNG)

**Comment by MaynardMiner** [comment]
The issue is, is that Windows 7/8 has no PNPCLASS in WMI.

```
pwsh
Get-CimInstance -namespace root\cimv2 -class Win32_PnPEntity | where Service -Eq nvlddmkm,amdkmdap,igfx,BasicDisplay
```

In CMD.exe. See if that returns a list of all your cards. If it does, I can fix.

**Comment by MaynardMiner** [comment]
I believe it is now fixed, if you want to check, and download .zip again directly from repository.

**Comment by petrovsky34** [comment]
ok

**Comment by petrovsky34** [comment]
-command   -cuda 9.2 no work

**Comment by petrovsky34** [comment]
add
cuda 9.2---
                  windows version---
cuda 9.1---

**Comment by petrovsky34** [comment]
have a nice weekend)))

**Comment by MaynardMiner** [comment]
@petrovsky34 

-Cuda was for linux only. I stopped supporting it the moment HiveOS default cuda became 10. It was notated in a release awhile ago.

You can use ``version update`` command or manually paste new links into config\nvidia-win.json with different miner version with different cuda.

It is waaaaay too much work to support miners for both linux and Windows, on top of supporting multiple cuda. If I get time I will add, but it is low priority.

**Comment by petrovsky34** [comment]
when updating which files are affected??? I change and then I have to redo everything??!?

**Comment by MaynardMiner** [comment]
@petrovsky34 SWARM saves all configs in the config folder but the update\ folder (this is so I can update miners).

SWARM also saves all statistics in stats folder, as well as backup stats in backup folder.

Just save a copy of nvidia-win.json prior to updating, then paste it back in after updating.

**Comment by petrovsky34** [comment]
Thanks)))

**Comment by MaynardMiner** [comment]
@petrovsky34 I am curious- What command is this?

![image](https://user-images.githubusercontent.com/38481606/57265967-e9dbdd00-7047-11e9-9d0c-d20c7ce3f015.png)

**Comment by petrovsky34** [comment]
bat file


@cd /d %~dp0

@if not "%CUDA_DEVICE_ORDER%"=="PCI_BUS_ID" (setx CUDA_DEVICE_ORDER PCI_BUS_ID) > nul

@set "command=& .\listdevices.ps1"

@echo off

where pwsh.exe >nul 2>nul
if %errorlevel%==1 (
    powershell -version 5.0 -windowstyle maximized -executionpolicy Bypass -command "%command%"
    goto end
)
pwsh -windowstyle maximized -executionpolicy Bypass -command "%command%"

:end

pause

**Comment by petrovsky34** [comment]
from another miner

**Comment by MaynardMiner** [comment]
Note for users:

Attempted fix in version 2.2.5

**Comment by MaynardMiner** [comment]
I have heard no issues of it not working since 2.2.5. Closing.
